Lib/unittest/util.py

```diff
@@ -1,6 +1,6 @@
 """Various utility functions."""
 
-from collections import namedtuple, OrderedDict
+from collections import namedtuple
```

Comment:
The collections module ends up loaded anyway, so removing OrderedDict doesn’t really change anything for this module.  @rhettinger would you mind giving your opinion for this PR?